kubernetes-gcp-yaml/Pulumi.yaml

```
@@ -2,7 +2,7 @@ name: ${PROJECT}
 description: ${DESCRIPTION}
 runtime: yaml
 template:
-  description: A YAML program to create a Kubernetes cluster on Google Cloud
+  description: A YAML program to deploy a Kubernetes cluster on Google Cloud
```

It looks like everywhere else we changed to `Pulumi YAML`.

```suggestion
  description: A Pulumi YAML program to deploy a Kubernetes cluster on Google Cloud
```